### DISCUSSION TITLE
In EditCargo, 0 mass & area for mount and Mobile facility

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#3142] [Q&A]
There are some entries in the BD tab in the Cargo Editor window with value 0 on Mass and Area. Was wondering if this is WAD, if not I'll make a list and drop them. If it is, nothing said.
regards GJ
![Cargo0](https://user-images.githubusercontent.com/99532490/223210484-8c4d53eb-7e67-4cc8-8cc5-6a4ca6413c16.jpg)

**Comment by Parel803** [comment]
Two more questions about Cargo:
1: how do i now which ship can drop cargo/troops ashore like the polnochny, RHIB, LCU? Is that the type of ship unit?
2: how is the time of unloading calculated? Noticed the Polnochny took 4 hours to unload 1 pax. Is this a DB values?
best regards GJ

**Reply by PygmalionOfCyprus** [reply]
1. Ships in the "Amphibious" category are usually your best bet, I think there is also logic to allow either Small Watercraft or anything with a low enough draft to do it (which would enable RHIBs, etc.) but offhand I don't recall exactly how the small boats are handled.
2. This is determined code-side AFAIK but that definitely seems like a bug. I would report it in the Tech Support forums with a save, etc. so our cargo gurus can look at it.

**Comment by PygmalionOfCyprus** [comment]
> There are some entries in the BD tab in the Cargo Editor window with value 0 on Mass and Area. Was wondering if this is WAD, if not I'll make a list and drop them.

Looking at those particular entries, those all seem to be our generic placeholder facilities which are meant mostly for abstraction (comes in handy during some large pro events). That would be why they have silly cargo values.

**Comment by Parel803** [comment]
All understood, thx for your time.
